test_scripts/Policies/GetSystemTime/020_GetSystemTime_mobile_cer_becomes_not_valid_sdl_cer_becomes_valid_with_PTU.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile app starts secure RPC service
+-- 2) Mobile certificate is up to date and sdl certificate is not up to date
+-- 3) SDL requests GetSystemTime
+-- 4) According to time from GetSystemTime response mobile certificate becomes not valid and sdl certificate becomes valid
```

Comment:
@GetmanetsIrina According to dates of mobile certificate (17-22) and ```systemTime``` = 20 mobile certificate still valid